lib/active_merchant/billing/gateways/paymentez.rb

```diff
@@ -134,6 +134,7 @@ def add_customer_data(post, options)
         post[:user][:email] = options[:email]
         post[:user][:ip_address] = options[:ip] if options[:ip]
         post[:user][:fiscal_number] = options[:fiscal_number] if options[:fiscal_number]
+        post[:user][:phone] = (options[:billing_address][:phone] if options[:billing_address]) || (options[:phone])
```

Comment:
Looks good, but if we want the GSF to act as an override, it will need to come first in the `||` check.

Reply:
FYI, the Travis failure is due to a Rubocop problem with a file outside of this commit, being addressed elsewhere.

Reply:
It's been fixed; you should be good if you rebase. (I literally _just_ merged it; it's not like you missed it.)